code/game/machinery/_machinery.dm

```diff
@@ -180,8 +180,10 @@ Class Procs:
 /obj/machinery/proc/locate_machinery()
 	return
 
-/obj/machinery/process()//If you dont use process or power why are you here
-	return PROCESS_KILL
+/obj/machinery/process()
+	if(!use_power) //machines without process() set would bug and not draw idle, this checks if they draw power
+		return PROCESS_KILL
+	return
```

Comment:
this is doing nothing but wasting CPU power. If a machine isn't doing anything in process(), it should be returning PROCESS_KILL.

Reply:
everything uses static power now, thanks for the advice!